x/interchainquery/genesis_test.go

```diff
@@ -0,0 +1,89 @@
+package interchainquery_test
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	ibctesting "github.com/cosmos/ibc-go/v3/testing"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/persistenceOne/persistence-sdk/simapp"
+	"github.com/persistenceOne/persistence-sdk/x/interchainquery"
+	"github.com/persistenceOne/persistence-sdk/x/interchainquery/keeper"
+	"github.com/persistenceOne/persistence-sdk/x/interchainquery/types"
+)
+
+func init() {
+	ibctesting.DefaultTestingAppInit = simapp.SetupTestingApp
+}
+
+func TestInterChainQueryTestSuite(t *testing.T) {
+	suite.Run(t, new(InterChainQueryTestSuite))
+}
+
+type InterChainQueryTestSuite struct {
+	suite.Suite
+
+	coordinator *ibctesting.Coordinator
+
+	chainA *ibctesting.TestChain
+	chainB *ibctesting.TestChain
+	path   *ibctesting.Path
+}
+
+func (s *InterChainQueryTestSuite) GetSimApp(chain *ibctesting.TestChain) *simapp.SimApp {
```

Comment:
usualy we use `suite *InterChainQueryTestSuite` instead of `s`
Interchain query might do this, but we want to make sure our code base is first consistent with the rest of out own codebase. https://github.com/persistenceOne/persistence-sdk/blob/master/x/epochs/keeper/keeper_test.go#L17